docs-user/gitpod.md

```diff
@@ -0,0 +1,17 @@
+# Setting up profiler on gitpod
+
+Instead of configuring a local setup, you can also use [gitpod](https://www.gitpod.io/), an online continuous development environment  with minimum setup.
+To use gitpod, you can login using your gitHub account and then follow the steps below. 
+
+- Change the URL of your browser to the respository, pull request or issue you want to open on gitpod e.g. for the [profiler project](https://github.com/firefox-devtools/profiler), URL of the upstream repository is `https://github.com/firefox-devtools/profiler`. You can also use the forked repository if you wish to start contributing to the project. 
+- Prefix the URL in the address bar of your browser with `gitpod.io/#` e.g. `https://github.com/firefox-devtools/profiler` becomes `https://gitpod.io/#https://github.com/firefox-devtools/profiler`.
+- Gitpod will then  launch a workspace for you and clone the repository, branch, commit or pull request depending on the URL in the first step.
+- If you are using gitpod for the first time, you will have to authorize access to your GitHub account.This is necessary so you can access your data from within Gitpod.
+- To authorize access, click your avatar/profile picture at the top right hand section of the setup. A dropdown menu will be displayed. In the dropdown menu, click `Open Access Control`. It will open Access Control page from where you can check the checkboxes and click update. 
+- A popup box, "Open Ports View", also appears at the bottom right hand section of the setup with the message `A service is available on port 4242`. You can click `Open Preview` or `Open Browser` either of which will open [profiler.firefox.com](https://profiler.firefox.com/) for your setup. If you have closed "Open Ports View", you can display it by clicking "PORTS" at the right hand section of the bottom bar.
```

Comment:
same for this, and maybe add a title: `## Open the profiler UI in your web browser`

BTW when I tried it, `Open Preview` didn't work for me. Does it work for you? If this doesnt maybe we can write that `You can click Open Browser, this will open the profiler UI in a separate tab.`.

Reply:
> BTW when I tried it, Open Preview didn't work for me. Does it work for you? 

Didn't work for me either. I thought the problem was with my setup.